### PR TITLE
Identifiable extension - VoltageLevel + Connectable

### PR DESCRIPTION
**Pull request by colinepiloquet**
Signed-off-by: Coline Piloquet <coline.piloquet@rte-france.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
#1

### CODE REVIEW

File: iidm-extensions/src/main/java/com/powsybl/shortcircuits/extensions/VoltageLevelShortCircuit.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import com.powsybl.commons.extensions.Extension;
+import com.powsybl.iidm.network.VoltageLevel;
+
+/**
+ * @author Coline Piloquet <coline.piloquet@rte-france.com>
+ */
+public interface VoltageLevelShortCircuit extends Extension<VoltageLevel> {
+    @Override
+    default String getName() {
+        return "voltageLevelShortCircuits";
```
Comment (annetill) [inline iidm-extensions/src/main/java/com/powsybl/shortcircuits/extensions/VoltageLevelShortCircuit.java:19]: Can you remove the `s`?

File: iidm-extensions/src/main/java/com/powsybl/shortcircuits/extensions/VoltageLevelShortCircuit.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    /**
+     * Get minimum allowable peak short-circuit current
+     */
+    double getIpMin();
```
Comment (annetill) [inline iidm-extensions/src/main/java/com/powsybl/shortcircuits/extensions/VoltageLevelShortCircuit.java:25]: Unit missing.

File: iidm-extensions/src/main/java/com/powsybl/shortcircuits/extensions/VoltageLevelShortCircuit.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+    /**
+     * Set minimum allowable peak short-circuit current
+     */
+    VoltageLevelShortCircuit setIpMin (double ipMin);
+
+    /**
+     * Get maximum allowable peak short-circuit current
```
Comment (annetill) [inline iidm-extensions/src/main/java/com/powsybl/shortcircuits/extensions/VoltageLevelShortCircuit.java:33]: Unit missing.

File: iidm-extensions/src/main/java/com/powsybl/shortcircuits/extensions/VoltageLevelShortCircuitAdderImpl.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+import com.powsybl.commons.extensions.AbstractExtensionAdder;
+import com.powsybl.iidm.network.VoltageLevel;
+
+/**
+ * @author Coline Piloquet <coline.piloquet@rte-france.fr>
+ */
+public class VoltageLevelShortCircuitAdderImpl extends AbstractExtensionAdder<VoltageLevel, VoltageLevelShortCircuit>
+        implements VoltageLevelShortCircuitAdder {
+
+    double ipMin=0;
```
Comment (annetill) [inline iidm-extensions/src/main/java/com/powsybl/shortcircuits/extensions/VoltageLevelShortCircuitAdderImpl.java:18]: Zero is not good here. NaN is better. No do forget space between `=`.

**Review by annetill — CHANGES_REQUESTED** [review]

File: iidm-extensions/src/main/java/com/powsybl/shortcircuits/extensions/VoltageLevelShortCircuit.java (hunk 2 above)
Comment (annetill) [reply iidm-extensions/src/main/java/com/powsybl/shortcircuits/extensions/VoltageLevelShortCircuit.java:25]: I think that in the rest of the code, we use () and not []...

**Review by MioRtia — COMMENTED** [review]
This is a good PR! Only remark: if ipMin/ipMax are mandatory, a check should be added for both to ensure they are not `NaN`

**Pull request merged**